### PR TITLE
added comment

### PR DESCRIPTION
**Pull request by lhubbard-USGS**
hooray

### CODE REVIEW

File: R/calc_water_year.R [hunk 1]
```diff
@@ -16,4 +16,4 @@ calc_water_year <- function(date.vec){
   return(water.year)
 }
 
-#testing: conflict resolution
```
Comment (scworland) [inline R/calc_water_year.R]: testing this feature

File: R/calc_water_year.R [hunk 2]
```diff
@@ -16,4 +16,4 @@ calc_water_year <- function(date.vec){
...
\ No newline at end of file
+################
```
Comment (rbcarvin) [inline R/calc_water_year.R]: :zap: okey dokey

Comment (scworland) [inline R/calc_water_year.R]: still testing

**Review by scworland — APPROVED** [review]
test

**Review by scworland — APPROVED** [review]
test

**Pull request merged**